### PR TITLE
The File System Standard PR

### PR DESCRIPTION
**Pull request by annevk**
This turns File System Access into the File System Standard as outlined in https://github.com/whatwg/sg/issues/176.

Review from @mkruisselbrink and @domenic/@foolip appreciated.

***
<a href="https://whatpr.org/fs/1.html" title="Last updated on Dec 13, 2021, 1:24 PM UTC (8c671c2)">Preview</a> | <a href="https://whatpr.org/fs/1/840b322...8c671c2.html" title="Last updated on Dec 13, 2021, 1:24 PM UTC (8c671c2)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1871,136 +980,14 @@ invoked, must run these steps:
...
-potentially really large offset past the end of a file makes it much easier and lower cost to
-create large files, on most file systems such files should not actually take up as much disk space as
-most commonly used file systems support sparse files (and thus wouldn't actually store the NUL
-bytes generated by resizing a file or seeking past the end of it).
 
-Whatever mitigations user agents use to guard against websites filling up a disk via either
-quota managed storage or the existing downloads mechanism should also be employed when websites
-use this API to write to disk.
+<p boilerplate=ipr>This Living Standard includes material copied from W3C WICG’s
+<a href=https://wicg.github.io/file-system-access/>File System Access</a>, which is available under
```
Comment (domenic) [inline index.bs:991]: Add `<cite>`?

File: index.bs [hunk 2]
```diff
@@ -1871,136 +980,14 @@ invoked, must run these steps:
...
-overhead). While the presence of {{FileSystemWritableFileStream/truncate()}} and writing at a
-potentially really large offset past the end of a file makes it much easier and lower cost to
-create large files, on most file systems such files should not actually take up as much disk space as
-most commonly used file systems support sparse files (and thus wouldn't actually store the NUL
-bytes generated by resizing a file or seeking past the end of it).
 
-Whatever mitigations user agents use to guard against websites filling up a disk via either
-quota managed storage or the existing downloads mechanism should also be employed when websites
-use this API to write to disk.
+<p boilerplate=ipr>This Living Standard includes material copied from W3C WICG’s
```
Comment (domenic) [inline index.bs:990]:
```suggestion
<p boilerplate=ipr>This Living Standard includes material copied from W3C WICG's
```

**Review by domenic — COMMENTED** [review]
Repo structure stuff looks good. I skimmed index.bs changes but probably @mkruisselbrink is the right reviewer for that.

**Review by foolip — APPROVED** [review]

**Review by mkruisselbrink — APPROVED** [review]
Sorry for not getting to this sooner. This all looks pretty reasonable to me.

**Pull request merged**